public/modules/custom/helfi_ptv_integration/src/HelfiPTV.php

```diff
@@ -0,0 +1,345 @@
+            $hours['sv']['hours'] .= 'Ons ' . substr($day->from, 0, -3) . '-' . substr($day->to, 0, -3) . ' ';
+            break;
+          case $day->dayFrom === 'Thursday':
+            $hours['fi']['hours'] .= 'To ' . substr($day->from, 0, -3) . '-' . substr($day->to, 0, -3) . ' ';
+            $hours['en']['hours'] .= 'Thu ' . substr($day->from, 0, -3) . '-' . substr($day->to, 0, -3) . ' ';
+            $hours['sv']['hours'] .= 'Tor ' . substr($day->from, 0, -3) . '-' . substr($day->to, 0, -3) . ' ';
+            break;
+          case $day->dayFrom === 'Friday':
+            $hours['fi']['hours'] .= 'Pe ' . substr($day->from, 0, -3) . '-' . substr($day->to, 0, -3) . ' ';
+            $hours['en']['hours'] .= 'Fri ' . substr($day->from, 0, -3) . '-' . substr($day->to, 0, -3) . ' ';
+            $hours['sv']['hours'] .= 'Fre ' . substr($day->from, 0, -3) . '-' . substr($day->to, 0, -3) . ' ';
+            break;
+          case $day->dayFrom === 'Saturday':
+            $hours['fi']['hours'] .= 'La ' . substr($day->from, 0, -3) . '-' . substr($day->to, 0, -3) . ' ';
+            $hours['en']['hours'] .= 'Sat ' . substr($day->from, 0, -3) . '-' . substr($day->to, 0, -3) . ' ';
+            $hours['sv']['hours'] .= 'Lör ' . substr($day->from, 0, -3) . '-' . substr($day->to, 0, -3) . ' ';
+            break;
+          case $day->dayFrom === 'Sunday':
+            $hours['fi']['hours'] .= 'Su ' . substr($day->from, 0, -3) . '-' . substr($day->to, 0, -3) . ' ';
+            $hours['en']['hours'] .= 'Sun ' . substr($day->from, 0, -3) . '-' . substr($day->to, 0, -3) . ' ';
+            $hours['sv']['hours'] .= 'Sön ' . substr($day->from, 0, -3) . '-' . substr($day->to, 0, -3) . ' ';
+            break;
+        }
+      }
+      if (isset($fieldStrings)) {
+        $fieldStrings['fi'] .= $hours['fi']['additional'] . ' ' . $hours['fi']['hours'];
+        $fieldStrings['en'] .= $hours['en']['additional'] . ' ' . $hours['en']['hours'];
+        $fieldStrings['sv'] .= $hours['sv']['additional'] . ' ' . $hours['sv']['hours'];
+      } else {
+        $fieldStrings['fi'] = $hours['fi']['additional'] . ' ' . $hours['fi']['hours'];
+        $fieldStrings['en'] = $hours['en']['additional'] . ' ' . $hours['en']['hours'];
+        $fieldStrings['sv'] = $hours['sv']['additional'] . ' ' . $hours['sv']['hours'];
+      }
+
+    }
+
+    return isset($fieldStrings) ? $fieldStrings : '';
+  }
+
+  private function getPhoneNumbers($phoneNumbers) {
+    $phoneData = [];
+    foreach ($phoneNumbers as $phoneNumber) {
+      if ($phoneNumber->type !== 'Phone') {
+        continue;
+      }
+      $additional = $phoneNumber->additionalInformation !== null ? $phoneNumber->additionalInformation . ' ' : '';
+      $phoneData[$phoneNumber->language][] = $additional . $phoneNumber->prefixNumber . $phoneNumber->number;
+    }
+    return $phoneData;
+  }
+
+  private function getEmails ($emailAddresses) {
```

Comment:
Let's start to use return type declarations for functions. e.g. `getEmails($emailAddresses): array {`